Orange/widgets/classify/owsvmclassification.py

```diff
@@ -175,9 +175,9 @@ def _add_type_box(self):
 
     def _add_optimization_box(self):
         super()._add_optimization_box()
-        gui.spin(self.optimization_box, self, "max_iter", 0, 1e6, 100,
+        gui.spin(self.optimization_box, self, "max_iter", 1, 1e6, 100,
```

Comment:
1 is not a good lower limit: if the user goes down and up again, he has 101, 201 ... I would suggest 50 as the lower limit and the step, that is, 

    gui.spin(self.optimization_box, self, "max_iter", 50, 1e6, 50,

Alternative would be 20 as the minimum and the step.